### DISCUSSION TITLE
3.0.0rc4

### DISCUSSION THREAD

**Discussion by michael-s-molina** [apache/superset#25283] [General]
Subject: [VOTE] Release Apache Superset 3.0.0 based on Superset 3.0.0rc4

Hello Superset Community,

This is a call for the vote to release Apache Superset version 3.0.0.

The release candidate:
https://dist.apache.org/repos/dist/dev/superset/3.0.0rc4/

Git tag for the release:
https://github.com/apache/superset/tree/3.0.0rc4

The Change Log for the release:
https://github.com/apache/superset/blob/3.0.0rc4/CHANGELOG.md

The Updating instructions for the release:
https://github.com/apache/superset/blob/3.0.0rc4/UPDATING.md

Public keys are available at:
https://www.apache.org/dist/superset/KEYS

The vote will be open for at least 72 hours or until the necessary number
of votes are reached.

Please vote accordingly:

[ ] +1 approve
[ ] +0 no opinion
[ ] -1 disapprove with the reason

Thanks,
The Apache Superset Team

<hr /><em>This discussion was created from the release <a href='https://github.com/apache/superset/releases/tag/3.0.0rc4'>3.0.0rc4</a>.</em>

**Comment by sfirke** [comment]
There's an issue with CTE queries, I haven't tested it in rc4 it seems to be caused by a PR that's in 3.0.0 so I would suspect it's relevant: https://github.com/apache/superset/issues/25278.

**Reply by michael-s-molina** [reply]
Thanks @sfirke. As soon as it's fixed we can cherry-pick it into 3.0.X. I don't view it as a blocker for the release.

**Comment by zhaorui2022** [comment]
Hi. We ran into this issue which seems also happening in the older version (we tested in 2.0.0) 
https://github.com/apache/superset/issues/25289

**Reply by michael-s-molina** [reply]
Thanks @zhaorui2022. I tagged @justinpark and we'll cherry-pick it into 3.0.X.

**Comment by Chaifist** [comment]
Hi, in v.3.0.0rc4 problem with export data to Excel with column datetime with time zone. issues in 2.1.0 [#23584](https://github.com/apache/superset/issues/23584)

**Reply by michael-s-molina** [reply]
Thanks @Chaifist. I tagged @betodealmeida and when a fix is provided we'll cherry-pick it into 3.0.X .

**Comment by michael-s-molina** [comment]
Closing this a RC4 passed! I'll cherry-pick open bugs in the next patch release. Big thanks to all of you ❤️ !
